composer.json

```diff
@@ -92,7 +92,7 @@
         "pda/pheanstalk": "^4.0",
         "phpstan/phpstan": "^1.4.7",
         "phpunit/phpunit": "^9.5.8",
-        "predis/predis": "^1.1.9",
+        "predis/predis": "^2.0",
```

Comment:
`^1.1.9|^2.0`, so prefer lowest gets 1.x?

Reply:
I'm just experimenting with this PR to force 2.0. I'll revert that.